### ISSUE TITLE
Updates in the user interface.

### ISSUE THREAD

**Issue by Raza403** [fulldecent/moodle-local_enrollment_tokens#23]
index.php
<img width="1440" alt="Screenshot 2024-10-24 at 7 14 11 PM" src="https://github.com/user-attachments/assets/1ec1a6d1-ada0-4a92-802b-eae9396881fe">

view_tokens.php
<img width="1440" alt="Screenshot 2024-10-24 at 7 17 20 PM" src="https://github.com/user-attachments/assets/7942b8e8-fa14-4cb8-afea-33e27a1329eb">

**Comment by fulldecent** [comment]
Hi @Raza403 what is there to do here?

**Comment by Raza403** [comment]
Yes, I am adding this to tomorrow's meeting. We can discuss updates in the UI.

**Comment by Raza403** [comment]
The user interface is updated, we have separate issues open for this. Closing this issue now.
